mlir/test/Dialect/Affine/SuperVectorize/vectorize_affine_apply.mlir

```diff
@@ -0,0 +1,65 @@
+// RUN: mlir-opt %s -affine-super-vectorize="virtual-vector-size=8 test-fastest-varying=0" -split-input-file | FileCheck %s
+
```

Comment:
Please check this example, I see it crashing in my local setup with your patch:
```mlir
func.func @crash(%arg0: memref<8x12x16xf32>, %arg1: memref<8x24x48xf32>) {
  affine.for %arg2 = 0 to 8 {
    affine.for %arg3 = 0 to 12 {
      affine.for %arg4 = 0 to 48 {
        %1 = affine.apply affine_map<(d0) -> (d0 mod 16)>(%arg4)
        %2 = affine.load %arg0[%arg2, %arg3, %1 + 1] : memref<8x12x16xf32>
        affine.store %2, %arg1[%arg2, %arg3, %arg4] : memref<8x24x48xf32>
      }
    }
  }
  return
}
```

Reply:
In this case, we already have `affine.apply` to calculate the index `%1`. `affine.load` still uses another expression to update its third index. `makePermutationMap` will detect the duplication and file an assert. If we move `+ 1` into `affine.apply`, it can generate code correctly.